docs/F-Refinements/string-library.md

```diff
@@ -212,27 +220,31 @@ For example:
 
 int main(void)
 {
-        int i, len;
-        char surname[31], fullName[62]; 
-
-        printf("Given name  : ");
-        scanf("%30[^\n]%*c", fullName);
-        printf("Surname     : ");
-        scanf("%30[^\n]%*c", surname);
-        strcat(fullName, " ");
-        strcat(fullName, surname);
-        printf("Full name is %s\n", fullName); 
-
-        return 0;
+    int i, len;
+    char surname[31], fullName[62];
```

Comment:
The full Name sounds like surname + given name.
Just to lessen the confusion, how about if we put `givenName` instead of `fullName`
And I think the size of the array can be just the same as the size of the surname array.

Reply:
I don't think so. This is the example for `strcat()` which concatenates `surname` and `givenname` to `fullname` so the `fullname`  arr's size should always be greater than the `surname` arr's size.